ad9361/sw/Makefile.generic

```diff
@@ -13,7 +13,7 @@ EXEC=ad9361_generic
 all: $(SOURCES) $(EXEC)
 
 $(EXEC): libad9361.a $(MAIN_SOURCES)
-	$(CC) $(LDFLAGS) $(MAIN_SOURCES) -o $@ -I$(PLATFORM) -lad9361 -L. -Wl,--gc-sections
+	$(CC) $(LDFLAGS) $(MAIN_SOURCES) -o $@ -I$(PLATFORM) -lad9361 -L. -Wl,-dead_strip
```

Comment:
Does `dead_strip` work on Linux?

Reply:
Since the generic build is made also on linux without errors, I suppose yes.

Reply:
You need to add a `make clean` to the travis script after the Makefile.linux build. Otherwise it doesn't try to re-build the files.

Reply:
I get this:

> cc  main.c  -o ad9361_generic -Iplatform_generic -lad9361 -L. -Wl,-dead_strip
> /usr/bin/ld: warning: cannot find entry symbol ad_strip; defaulting to 00000000000008f0

Reply:
Probably something like this https://github.com/RIOT-OS/RIOT/pull/5794/commits/291c1a54ebfde8e5d7ff3db97d5b01befae3b7f0